src/dataflow/core/linearize.py

```diff
@@ -58,17 +59,6 @@ def sexp_deformatter(sexp_tokens: List[str]) -> List[str]:
     """
     sexp_str = " ".join(sexp_tokens)
 
-    sexp_str = re.sub(
```

Comment:
I think we might never have had real JSON anyways.

Reply:
to make sure I'm on the same page -- it seems that we do not care backward compatibility, i.e., the code would broken on the SMCalFlow 1.0 data?
If so, probably we should change the README.md since the instructions there would be broken.

Reply:
We do, I think all the changes I'm making can read in the old format. There is no JSON in CalFlow. 
```(base) jajacMScomp:metals adpauls$ grep "schema" ~/sm/calflow/train.dataflow_dialogues.jsonl
(base) jajacMScomp:metals adpauls$ grep "underlying" ~/sm/calflow/train.dataflow_dialogues.jsonl```

Reply:
I see. Maybe there are some dead code [here](https://github.com/microsoft/task_oriented_dialogue_as_dataflow_synthesis/blob/ab0a6d65ead8edfea1624bf0051ccab72c394f51/src/dataflow/core/lispress.py#L267).

It seems that removing the `re.sub`, the `sexp_deformatter` is trivial. Maybe we do not need this method anymore?
Can you also remove the `re.sub` in `sexp_formatter` (or completely delete these two methods)?